material/problems/VectorGeometry/FirstAndTen/FirstAndTen.tex

```diff
@@ -1,116 +1,240 @@
-\usepackage{import}
-
-% This file must be found via the TEXINPUTS environment variable.
-\input{handout_helpers.tex}
-
-\pagestyle{plain}
-\pagenumbering{arabic}
-
-\begin{document}
-\HandoutHeader{Vector Equations and Geometry}
-%
-\vspace{-0.5pc}
-\small
-%
-\begin{enumerate}
-\item \textColorBold{darkerBlue}{Cameras and Robotics in the NFL}
-\\[0.5pc]
+%   File: FirstAndTen.tex
+% Author: Adam Leeper
+%------------------------------------------------------------------------------
+%\\[0.45pc]
+\providecommand{\isolatedBuild}[1]{#1}% Fallback definition to build normally.
+\isolatedBuild{
+  \documentclass[11pt,letterpaper]{book}
+  \input{handout_helpers.tex}   % Found via TEXINPUTS environment variable.
+  \isolatedBuildHeader{Vector Equations and Geometry}
+                      {Cameras and Robotics in the NFL}
 }
+%%%
+%%%
+%%%
+\newcommand\MyLBrace[2]{%
+  \left.\rule{0pt}{#1}\right\}\text{#2}}
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %
 \begin{minipage}[t]{0.7\linewidth}
-The 1st and Ten Graphics System is used to  show a virtual yellow line on video footage of football games.
-The system continuously measures the position and orientation of the camera relative to the the field. Hence, position information from the field can be converted into the frame of the camera for overlaying on the video in real-time, even as the camera rotates.
-%When an operator clicks on the image, the click location is converted into a position on the field.
-\\[0.45pc]
-Let the field be a rigid frame \basis{N} with orthogonal unit
-vectors \uvecBasisxyz{n} oriented as shown.
-Let point $N_o$ be fixed in \basis{N} at zero height above the field.
-Assume the field lies in the \uvecxy{n} plane.
-%
-\\[0.45pc]
-Let the camera body be a rigid frame \basis{B} with orthogonal
-unit vectors \uvecBasisxyz{b} and a point $B_o$ fixed in \basis{B}.
-Assume the position vector from $N_o$ to $B_o$ is known to be $\posvec{N_o}{B_o} = (50\mathrm{~yards})~\uvecx{n} + (20\mathrm{~yards})~\uvecz{n}$.
-%
-\\[0.45pc] Finally, let the football be identified as point $Q$.
-
+  The 1st and Ten Graphics System is used to  show a virtual yellow line on
+  video footage of football games.The system continuously measures the position
+  and orientation of the camera relative to the the field. Hence, position
+  information from the field can be converted into the frame of the camera for
+  overlaying on the video in real-time, even as the camera rotates.
+  %When an operator clicks on the image, the click location is converted into a
+  %position on the field.
+  \\[0.45pc]
+  Let the field be a rigid frame \basis{N} with orthogonal unit
+  vectors \uvecBasisxyz{n} oriented as shown.
+  Let point $N_o$ be fixed in \basis{N} at zero height above the field.
+  Assume the field lies in the \uvecxy{n} plane.
+  %
+  \\[0.45pc]
+  Let the camera body be a rigid frame \basis{B} with orthogonal
+  unit vectors \uvecBasisxyz{b} and a point $B_o$ fixed in \basis{B}.
```

Comment:
I recently started using an `\origin` macro. So, replace instances of B_o, N_o, etc with, for example, `\origin{B}`.